sdk/communication/azure-communication-phonenumbers/CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ Python 2.7 is no longer supported. Please use Python version 3.6 or later.
 
 - Users can now purchase United Kingdom (GB) toll free and geographic phone numbers for PSTN Calling
 - Users can now purchase Denmark (DK) toll free and geographic phone numbers for PSTN Calling
+- Added support for proactive refreshing of tokens
```

Comment:
Since this version is already released, I believe this should be part of a new version `1.1.0b2`.